### ISSUE TITLE
Low results

### ISSUE THREAD

**Issue by UnrealPawn** [lulujianjie/person-reid-tiny-baseline#1]
Nice work!
But I got low results on market dataset using your default config. (I see you get 88% Rank1)
```
The test feature is normalized
=> Computing DistMat with euclidean distance
2019-06-13 09:21:40,468 baseline-person-reid.train INFO: Validation Results - Epoch: 120
2019-06-13 09:21:40,468 baseline-person-reid.train INFO: mAP: 57.9%
2019-06-13 09:21:40,468 baseline-person-reid.train INFO: CMC curve, Rank-1  :82.7%
2019-06-13 09:21:40,468 baseline-person-reid.train INFO: CMC curve, Rank-5  :93.4%
2019-06-13 09:21:40,468 baseline-person-reid.train INFO: CMC curve, Rank-10 :95.8%
```
any suggestion ?
I also tested on Duke dataset, but the Rank1 and mAP are still low.
```
The test feature is normalized
=> Computing DistMat with euclidean distance
2019-06-13 00:16:53,749 baseline-person-reid.train INFO: Validation Results - Epoch: 120
2019-06-13 00:16:53,750 baseline-person-reid.train INFO: mAP: 48.3%
2019-06-13 00:16:53,750 baseline-person-reid.train INFO: CMC curve, Rank-1  :70.3%
2019-06-13 00:16:53,750 baseline-person-reid.train INFO: CMC curve, Rank-5  :83.8%
2019-06-13 00:16:53,750 baseline-person-reid.train INFO: CMC curve, Rank-10 :87.0%

```

**Comment by lulujianjie** [comment]
My test result is as following:

> 2019-06-11 15:28:09,340 baseline-person-reid.test INFO: Enter inferencing
2019-06-11 15:30:27,886 baseline-person-reid.test INFO: Validation Results
2019-06-11 15:30:27,886 baseline-person-reid.test INFO: mAP: 70.5%
2019-06-11 15:30:27,886 baseline-person-reid.test INFO: CMC curve, Rank-1  :88.2%
2019-06-11 15:30:27,886 baseline-person-reid.test INFO: CMC curve, Rank-5  :95.7%
2019-06-11 15:30:27,886 baseline-person-reid.test INFO: CMC curve, Rank-10 :97.5%

And I have uploaded the pretrained weight [link](https://pan.baidu.com/s/1FrEOT3h7lAePddFHNWIEjg) with extraction code 

> u3q5

It is not supposed to be so low on Market1501. This repo can reproduce the results as good as official code on Market1501 when the input size is set to 256x128. Did you use the imagenet-pretrained ResNet model to initialize the backbone?
And I do not test it on Duke dataset yet.

**Comment by swg209** [comment]
@UnrealPawn  I run the test.py on market1501 with pretrained weight provided in this github, but also get a close result to yours.   My input size is 128x64.   Is there any explainment?
`Loading pretrained ImageNet model......`
`2019-07-25 11:13:55,222 baseline-person-reid.test INFO: Enter inferencing`
`The test feature is normalized`
`=> Computing DistMat with cosine similarity`
`2019-07-25 11:17:40,763 baseline-person-reid.test INFO: Validation Results`
`2019-07-25 11:17:40,763 baseline-person-reid.test INFO: mAP: 58.0%`
`2019-07-25 11:17:40,763 baseline-person-reid.test INFO: CMC curve, Rank-1  :82.1%`
`2019-07-25 11:17:40,764 baseline-person-reid.test INFO: CMC curve, Rank-5  :93.3%`
`2019-07-25 11:17:40,764 baseline-person-reid.test INFO: CMC curve, Rank-10 :95.7%`

**Comment by derkbreeze** [comment]
@swg209 Using only softmax loss without label smooth I obtained mAP 81.8% and Rank-1 93.3, Rank-5 97.4, Rank-10 98.4 with input size 128x64
